electric-motor-torque-curve/main.js

```diff
@@ -1,15 +1,16 @@
-// const selectElement = document.getElementById('.calc-1');
-// selectElement.addEventListener('change', (event) => {
-//   const result = document.querySelector('.result');
-//   result.textContent = `You like ${event.target.value}`;
-// });
-
 //////////////////////////////////////////////////
 // 0. Preliminary Stuff
 //////////////////////////////////////////////////
-/// Make sure that the files actually SEE each other
-//alert("Hello World!");
 
+  
+// Test URLs for loading URL Parameters  
+// file:///Users/asci-blue/Workspace/webtools/electric-motor-torque-curve/index.html?voltage-rated=1&voltage-supplied=1&no-load-torque=1&rated-torque=1&no-load-speed=1&rated-speed=1&no-load-current=1&rated-current=1
+
+// file:///Users/asci-blue/Workspace/webtools/electric-motor-torque-curve/index.html?voltage-rated=67&voltage-supplied=134&no-load-torque=312&rated-torque=3&no-load-speed=12&rated-speed=1&no-load-current=1&rated-current=1
+
+// file:///Users/asci-blue/Workspace/webtools/electric-motor-torque-curve/index.html?voltage-rated=67&voltage-supplied=1&no-load-torque=1&rated-torque=3&no-load-speed=1&rated-speed=1&no-load-current=1&rated-current=1
+
+// file:///Users/asci-blue/Workspace/webtools/electric-motor-torque-curve/index.html?voltage-rated=1&voltage-supplied=1&no-load-torque=1&rated-torque=3&no-load-speed=1&rated-speed=1&no-load-current=1&rated-current=1
```

Comment:
Remove personal information from these comments. These comments include your computers local directory structure. You only need to keep the stuff after the ? Mark. Write a short blurb about how to copy this to the URL and what values these constitute. Would also be great if we had the output values but you can leave those as TBD for now.

Add an tech debt issue (make an issue on GitHub for this repo) stating that we need to add the expected outputs in these comments. Then In the code add

```
// TODO(#<issue number>): add output values to comments later
```